cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -195,13 +195,17 @@ public static Secret buildSecret(ClusterCa clusterCa, Secret secret, String name
                 CertAndKey eoCertAndKey = clusterCa.generateSignedCert(commonName, Ca.IO_STRIMZI);
                 data.put(keyCertName + ".key", eoCertAndKey.keyAsBase64String());
                 data.put(keyCertName + ".crt", eoCertAndKey.certAsBase64String());
+                data.put(keyCertName + ".p12", eoCertAndKey.keyStoreAsBase64String());
+                data.put(keyCertName + ".password", eoCertAndKey.storePasswordAsBase64String());
             } catch (IOException e) {
                 log.warn("Error while generating certificates", e);
             }
             log.debug("End generating certificates");
         } else {
             data.put(keyCertName + ".key", secret.getData().get(keyCertName + ".key"));
             data.put(keyCertName + ".crt", secret.getData().get(keyCertName + ".crt"));
+            data.put(keyCertName + ".p12", secret.getData().get(keyCertName + ".p12"));
```

Comment:
Just to double check ... how does this handle the upgrade? Is it done somewhere else?

Reply:
good catch, I have to fix it